tests/scripts/test_pkgdev_mask.py

```diff
@@ -199,3 +199,32 @@ def test_last_rites(self):
                 cat/pkg
             """)
             self.masks_path.write_text("")  # Reset the contents of package.mask
+
+    def test_mask_bugs(self):
+        removal_date = self.today + timedelta(days=30)
+        today = self.today.strftime('%Y-%m-%d')
+        removal = removal_date.strftime('%Y-%m-%d')
+        for bugs in (['-b', '42'], ['--bug', '42'], ['-b', '42', '-b', '43'], ['--bug', '42', '--bug', '43']):
+            with os_environ(EDITOR="sed -i '1s/$/mask comment/'"), \
+                    patch('sys.argv', self.args + ['cat/pkg'] + bugs), \
+                    pytest.raises(SystemExit), \
+                    chdir(pjoin(self.repo.path)):
+                self.script()
+
+            bug_nums = bugs[1::2]
```

Comment:
This and the other bugs specific output related lines could be combined into the loop as mentioned above as expected output then it would be just a single var replacement in the format string.